some broken links in release checklist

they should refer to the proper child release checklist

Is this what you're referring to?
`|    Compiling apollo-spaceport v0.14.0 (https://github.com/apollographql/router.git?tag=v0.14.0#9c0864ca)
| The following warnings were emitted during compilation:
| 
| warning: Updating router-bridge
| warning: running `npm ci`
| 
| error: failed to run custom build command for `router-bridge v0.1.0 (https://github.com/apollographql/federation-rs.git?branch=geal/defer#4cdcd574)`
| 
| Caused by:
|   process didn't exit successfully: `/home/imann/code/Work180/graphql/target/release/build/router-bridge-5d43838979a9e279/build-script-build` (exit status: 101)
|   --- stdout
|   cargo:rerun-if-changed=js-src
|   cargo:warning=Updating router-bridge
|   cargo:warning=running `npm ci`
| 
|   --- stderr
|   npm ERR! code E404
|   npm ERR! 404 Not Found - GET https://pkg.csb.dev/apollographql/federation/commit/278029dd/@apollo/federation-internals
|   npm ERR! 404 
|   npm ERR! 404  '@apollo/federation-internals@https://pkg.csb.dev/apollographql/federation/commit/278029dd/@apollo/federation-internals' is not in this registry.
|   npm ERR! 404 
|   npm ERR! 404 Note that you can also install from a
|   npm ERR! 404 tarball, folder, http url, or git url.
| 
|   npm ERR! A complete log of this run can be found in:
|   npm ERR!     /root/.npm/_logs/2022-08-09T04_51_31_557Z-debug-0.log
|   thread 'main' panicked at 'assertion failed: Command::new(&npm).current_dir(&current_dir).args(&[\"ci\"]).status().unwrap().success()', /root/.cargo/git/checkouts/federation-rs-edb4f8c0f5624a5c/4cdcd57/federation-2/router-bridge/build.rs:41:9
|   note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
| warning: build failed, waiting for other jobs to finish...`

--- a/RELEASE_CHECKLIST.md
+++ b/RELEASE_CHECKLIST.md
@@ -9,8 +9,8 @@ The following table describes the packages that will be published by following a
 | package/binary | version | release checklist |
 | --- | --- | --- |
 | `apollo-federation-types` | any | [`./apollo-federation-types`](./apollo-federation-types/RELEASE_CHECKLIST.md) |
-| `harmonizer` | v0.x.x | [`./harmonizer-0`](./harmonizer-0/RELEASE_CHECKLIST.md) |
-| `harmonizer` | v2.x.x | [`./harmonizer-2`](./harmonizer-2/RELEASE_CHECKLIST.md) |
-| `router-bridge` | any | [`./router-bridge`](./router-bridge/RELEASE_CHECKLIST.md) |
-| `supergraph` | v0.x.x | [`./harmonizer-0`](./harmonizer-0/RELEASE_CHECKLIST.md) |
-| `supergraph` | v2.x.x | [`./harmonizer-2`](./harmonizer-2/RELEASE_CHECKLIST.md) |
+| `harmonizer` | v0.x.x | [`./harmonizer-0`](./federation-1/harmonizer/RELEASE_CHECKLIST.md) |
+| `harmonizer` | v2.x.x | [`./harmonizer-2`](./federation-2/harmonizer/RELEASE_CHECKLIST.md) |
+| `router-bridge` | any | [`./router-bridge`](./federation-2/router-bridge/RELEASE_CHECKLIST.md) |
+| `supergraph` | v0.x.x | [`./harmonizer-0`](./federation-1/harmonizer/RELEASE_CHECKLIST.md) |
+| `supergraph` | v2.x.x | [`./harmonizer-2`](./federation-2/harmonizer/RELEASE_CHECKLIST.md) |

